lib/absinthe/schema/notation/sdl_render.ex

```diff
@@ -0,0 +1,394 @@
+      arguments(directive.arguments),
+      " on ",
+      join(locations, " | ")
+    ])
+    |> description(directive.description)
+  end
+
+  defp render(%Blueprint.TypeReference.Name{name: name}) do
+    string(name)
+  end
+
+  defp render(%Blueprint.TypeReference.List{of_type: of_type}) do
+    concat(["[", render(of_type), "]"])
+  end
+
+  defp render(%Blueprint.TypeReference.NonNull{of_type: of_type}) do
+    concat([render(of_type), "!"])
+  end
+
+  defp render(:integer), do: "Int"
+
+  defp render(type) when is_atom(type) and type in @builtin_scalars do
+    type |> to_string |> String.capitalize()
+  end
+
+  defp render_list(items, seperator \\ line()) do
+    splitter =
+      items
+      |> Enum.any?(&(&1.description not in ["", nil]))
+      |> case do
+        true -> [nest(line(), :reset), line()]
+        false -> [seperator]
+      end
+
+    items
+    |> Enum.reverse()
+    |> Enum.reduce(:start, fn
+      item, :start -> render(item)
+      item, acc -> concat([render(item)] ++ splitter ++ [acc])
+    end)
+  end
+
+  defp arguments([]) do
+    empty()
+  end
+
+  defp arguments(args) do
+    any_descriptions? = Enum.any?(args, & &1.description)
+
+    group(
+      glue(
+        nest(
+          multiline(
+            glue(
+              "(",
+              "",
+              render_list(args, ", ")
+            ),
+            any_descriptions?
+          ),
+          2,
+          :break
+        ),
+        "",
+        ")"
+      )
+    )
+  end
+
+  defp default(nil) do
+    empty()
+  end
+
+  defp default(default_value) do
+    concat([" = ", to_string(default_value)])
+  end
+
+  defp deprecated(docs, nil) do
+    docs
+  end
+
+  defp deprecated(docs, %{reason: nil}) do
+    space(docs, "@deprecated")
+  end
+
+  defp deprecated(docs, %{reason: reason}) do
+    concat([
+      space(docs, "@deprecated"),
+      "(",
+      "reason: ",
+      deprecated_reason(reason),
+      ")"
+    ])
+  end
+
+  defp deprecated_reason(reason) do
+    reason
+    |> String.trim()
+    |> String.split("\n")
+    |> case do
+      [reason] ->
+        concat([~s("), reason, ~s(")])
+
+      reason_lines ->
+        concat(
+          nest(
+            block_string([~s(""")] ++ reason_lines),
+            2,
+            :always
+          ),
+          concat(line(), ~s("""))
+        )
+    end
+  end
+
+  defp description(docs, nil) do
+    docs
+  end
+
+  defp description(docs, description) do
+    description
+    |> String.trim()
+    |> String.split("\n")
+    |> case do
+      [description] ->
+        [~s("), description, ~s("), line(), docs]
+
+      description_lines ->
+        [block_string([~s(""")] ++ description_lines ++ [~s(""")]), line(), docs]
+    end
+    |> concat()
+  end
+
+  defp implements([]) do
+    empty()
+  end
+
+  defp implements(interface_types) do
+    interface_names = Enum.map(interface_types, & &1.name)
+
+    concat([
+      " implements ",
+      join(interface_names, ", ")
```

Comment:
```suggestion
      join(interface_names, " & ")
```

I think a `&` is used here https://graphql.github.io/graphql-spec/June2018/#ImplementsInterfaces

Reply:
thanks!